ros2topic/ros2topic/verb/echo.py

```diff
@@ -76,6 +77,12 @@ def add_arguments(self, parser, cli_name):
             '--no-arr', action='store_true', help="Don't print array fields of messages")
         parser.add_argument(
             '--no-str', action='store_true', help="Don't print string fields of messages")
+        parser.add_argument(
+            '--once', action='store_true', help="Print the first message received and then exit")
+        parser.add_argument(
+            '--timeout', type=unsigned_int, default=None,
+            help='Number of seconds before exiting (default: None)')
```

Comment:
@craigh92 no need to specify defaults, `ros2cli` parser should.

Reply:
Fixed :)